lib/validation.ex

```diff
@@ -1,9 +1,13 @@
 defmodule Uploadex.Validation do
   @moduledoc false
 
+
   @type file :: map() | String.t
```

Comment:
This `@type` is unnecessary, right? Since `file` is now `wrapped_file`

Reply:
The `file()` is used in the `wrapped_file` typespec definition

Reply:
Aah got it!